addons/account/models/account_move.py

```diff
@@ -20,6 +21,31 @@ class AccountMove(models.Model):
     _description = "Account Entry"
     _order = 'date desc, id desc'
 
+    @api.multi
+    @api.constrains('name', 'journal_id')
+    def _check_unique_sequence_number(self):
```

Comment:
Use the fixed version of the constraint:
https://github.com/odoo/odoo/blob/saas-12.4/addons/account/models/account_move.py#L1250

Reply:
I've just updated the commit.